test/e2e/config/caren.yaml

```diff
@@ -161,6 +184,19 @@ variables:
   AMI_LOOKUP_FORMAT: "konvoy-ami-{{.BaseOS}}-release-?{{.K8sVersion}}-*"
   AMI_LOOKUP_BASEOS: "rocky-9.1"
   AMI_LOOKUP_ORG: "999867407951"
+  NUTANIX_ENDPOINT: ""
+  NUTANIX_USER: ""
+  NUTANIX_PASSWORD: ""
+  NUTANIX_INSECURE: false
+  NUTANIX_PORT: 9440
+  CONTROL_PLANE_ENDPOINT_IP: ""
+  CONTROL_PLANE_ENDPOINT_PORT: 6443
+  NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: ""
+  NUTANIX_SUBNET_NAME: ""
+  DOCKER_HUB_USERNAME: ""
+  NUTANIX_PRISM_ELEMENT_ENDPOINT: ""
+  NUTANIX_STORAGE_CONTAINER_NAME: ""
```

Comment:
For n👀bs like me, could you please add comments to each of the Nutanix specific variables?

Reply:
For all the empty vars should they be here at all? In CI they can come for GHA Secrets, and running locally they would come from local envs.

Reply:
then how do you intend to convey what variables to set for a user in general. moreover we might add test specific variables too sometime. currently many a times we find out that certain variable is not set later from capi side. having a list is really useful. one can always override them